src/System.Net.Requests/src/System/Net/_TLSstream.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Security;
+using System.Net.Sockets;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace System.Net
+{
+    internal class TlsStream : NetworkStream
```

Comment:
The main issue here is that the rest of the code expects a NetworkStream, and SslStream does not derive from NetworkStream.  So TlsStream now is basically just a wrapper of SslStream that derives from NetworkStream.

We could fix this, but it would require some internal refactoring.